test/metabase/test/data/presto.clj

```diff
@@ -10,7 +13,7 @@
   (:import java.util.Date
            metabase.driver.presto.PrestoDriver))
 
-(resolve-private-vars metabase.driver.presto execute-presto-query! presto-type->base-type quote-name quote+combine-names)
+(resolve-private-vars metabase.driver.presto execute-presto-query! presto-type->base-type quote-name quote+combine-names time->str)
```

Comment:
Going forward it's preferable IMO to stop using `resolve-private-vars` and just do something like `#'presto/time->str` when you want to use something private in a test. The only reason I wrote `resolve-private-vars` in the first place was because I didn't realize you could do that

Also, don't forget to `require` `metabase.driver.presto`, otherwise `import` will potentially barf because Clojure doesn't know it needs to load a certain namespace for that class to get defined. It's working fine here because somebody else is implicitly loading it but relying on such things is sketchy

Reply:
Those sorts of errors usually end up popping up in confusing ways, like if you try to test a single namespace that ends up skipping loading all the other namespaces that normally implicitly load whatever it is you're expecting to be loaded

Reply:
Oops. I just noticed you didn't add this code and it was already like that... 😬 

Mind fixing it while you're here?

Reply:
Totally agree on the `metabase.driver.presto` thing and the resolve-private-vars I have always found confusing as the #' syntax is so easy. I'm 👍, I'll get it fixed up